Lexi.Core.Api.Tests.Unit/Services/Foundations/Feedbacks/FeedbackServiceTests.ValidationsAdd.cs

```diff
@@ -0,0 +1,88 @@
+﻿//=================================
+// Copyright (c) Tarteeb LLC.
+// Powering True Leadership
+//=================================
+
+using FluentAssertions;
+using Lexi.Core.Api.Models.Foundations.Feedbacks;
+using Lexi.Core.Api.Models.Foundations.Feedbacks.Exceptions;
+using Moq;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Lexi.Core.Api.Tests.Unit.Services.Foundations.Feedbacks
+{
+    public partial class FeedbackServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowValidationExceptionOnAddOfInputIsNullAndLogItAsync()
+        {
+            //given
+            Feedback nullFeedback = null;
+            var nullFeedbackException = new NullFeedbackException();
+
+            var expectedfeedbackValidationException =
+                new FeedbackValidationException(nullFeedbackException);
+
+            //when
+            ValueTask<Feedback> addFeedbackTask = this.feedbackService.AddFeedbackAsync(nullFeedback);
```

Comment:
@GolibjonTuxtamatov
@ZafarUrakov
Bu yerda this dan boshlab yangi qatorda bo'lganida yaxshiroq bo'lar edi